packages/kit-bg/src/dbs/local/LocalDbBaseContainer.ts

```
@@ -110,6 +113,20 @@ export abstract class LocalDbBaseContainer implements ILocalDBAgent {
     ttl: timerUtils.getTimeDurationMs({ seconds: 5 }),
   });
 
+  getAllRecordsByCache<T>(
+    cacheKey:
+      | 'allDbAccounts'
+      | 'allDbIndexedAccounts'
+      | 'allDbWallets'
+      | 'allDbDevices',
+  ) {
+    const allItemsInCache = this.dbAllRecordsCache.get(cacheKey) as T[];
+    if (allItemsInCache && allItemsInCache.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)